web/src/buddy_list.ts

```diff
@@ -62,6 +62,19 @@ function get_total_human_subscriber_count(
     return 0;
 }
 
+function get_non_participant_subscribers_count(
+    current_sub: StreamSubscription | undefined,
+    get_all_participant_ids: () => Set<number>,
+): number {
+    if (!current_sub) {
+        return 0;
+    }
+    const all_subs = new Set(peer_data.get_subscribers(current_sub.stream_id));
```

Comment:
Name this `all_subscribers`, to avoid confusion with `sub` often standing for `subscription`